standalone-metastore/pom.xml

```diff
@@ -206,12 +208,16 @@
         <artifactId>hadoop-mapreduce-client-core</artifactId>
         <version>${hadoop.version}</version>
       </dependency>
-      <!-- This is our one and only Hive dependency.-->
       <dependency>
         <groupId>org.apache.hive</groupId>
         <artifactId>hive-storage-api</artifactId>
         <version>${storage-api.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.hive</groupId>
```

Comment:
I dont think we want the hive-common to be included in dependencies in metastore, it should be the other way around where metastore dependencies are in hive, so you will want to move the relevant code to hive-standalone-metastore-common and then include it in relevant places in hive.